### ISSUE TITLE
rm before cp? 

### ISSUE THREAD

**Issue by Lexx3D** [systemik/Kobra3-Firmware#13]
Hey,

with the help of your repo i managed to get into the k3, thanks a lot. 
I had some minor issues which i successfully sorted out manually but finally: Job done. 

I might be wrong but isn't the rm deleting  the stuff  to copy? (cp source dest)

```
# clean old openssh
rm -rf /useremain/update_swu/openssh
# copy new openssh (with /var/empty changed to /tmp/empty in the binary)
cp -r /useremain/update_swu/openssh /useremain/openssh
```
Jay

**Comment by systemik** [comment]
Thanks for the info. You're right I messed up the folder names.
This is corrected with #14
I'll update the package soon.

**Comment by systemik** [comment]
Closed with #14
